vnext/template/cpp-lib/src/ReactNativeModule.h

```diff
@@ -0,0 +1,29 @@
+﻿#pragma once
+
+#include "JSValue.h"
+#include "NativeModules.h"
+
+// clang-format off
+
+using namespace winrt::Microsoft::ReactNative;
+
+namespace winrt::{{ namespaceCpp }}
+{
+
```

Comment:
add a comment with a link to an aka.ms pointing to the documentation for writing native modules in C++ (same for C# lib)